cmd/kubeadm/app/cmd/config.go

```diff
@@ -478,10 +479,11 @@ func NewCmdConfigImagesList(out io.Writer, mockK8sVersion *string) *cobra.Comman
 			kubeadmutil.CheckErr(err)
 			imagesList, err := NewImagesList(cfgPath, externalcfg)
 			kubeadmutil.CheckErr(err)
-			kubeadmutil.CheckErr(imagesList.Run(out))
+			kubeadmutil.CheckErr(imagesList.Run(out, outputFormat))
 		},
 	}
 	AddImagesCommonConfigFlags(cmd.PersistentFlags(), externalcfg, &cfgPath, &featureGatesString)
+	cmd.Flags().StringVar(&outputFormat, "output", "text", "Output format; available options are 'text', 'yaml' and 'json'")
```

Comment:
we should define i default flag for this in `options`.
we should also make `kubeadm version` use it (version already has machine readable output).

also version has the short version of `o`.